biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -135,18 +140,31 @@ export const SimulatePage = () => {
             <h2>Simulation Board</h2>
             <div className="simboard-controls">
               <button onClick={() => playSimulation()}>Play</button>
-              <button onClick={() => setCurrentStateFrame(0)}>Stop</button>
+              <button onClick={() => setCurrentStateFrame(0)}>Reset</button>
+            </div>
+            <div>
+              Time of day:{' '}
+              {currentStateFrame % 12 === 0 ? 12 : currentStateFrame % 12}:00{' '}
+              {currentStateFrame >= 12 ? 'PM' : 'AM'}
             </div>
             <SimulationBoard
               grid_height_km={simulationResults.townSize.height}
               grid_width_km={simulationResults.townSize.width}
               // TODO add changing indices to show the progression of time for each subsequent state
               // Find the GitHub issue: https://github.com/googleinterns/step141-2020/issues/64
               items={stateToGridItemRet(
-                simulationResults.states[currentStateFrame < simulationResults.states.length ? currentStateFrame : 0]
+                simulationResults.states[
+                  currentStateFrame < simulationResults.states.length
+                    ? currentStateFrame
+                    : 0
```

Comment:
when the simulation ends, wouldn't the simulation stop in the very last state, instead of state 0?

Reply:
It does,  I just added that as a safe guard. Should I delete?

Reply:
Ah thanks for clarifying, I think a safeguard is a great idea. You can add a comment saying `as safeguard, fallback to 0` or something similar if you want, but up to you